service/biz/enrollz_biz.go

```diff
@@ -27,13 +27,41 @@ import (
 	"google.golang.org/protobuf/encoding/prototext"
 )
 
+// Request to SwitchOwnerCaClient.IssueOwnerIakCert().
+type IssueOwnerIakCertReq struct {
```

Comment:
comment on exported type IssueOwnerIakCertReq should be of the form "IssueOwnerIakCertReq ..." (with optional leading article)

http://go/go-style/decisions#doc-comments

Here and all structs below

Reply:
Done! :)